Bottom: 6fd9cf80eaf68247cbeab15e65ea8c37ae4188b9
Top:    634f79679d597ae0dd889bb3a3e74f9e90f25382
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2015-01-12 01:16:05 +0000

Adjust STEP export to 2.3mm board thickness (Hack - need something quick!)


---

diff --git a/src/hid/step/step.c b/src/hid/step/step.c
index 9b775ac..d1b0452 100644
--- a/src/hid/step/step.c
+++ b/src/hid/step/step.c
@@ -31,7 +31,7 @@
 
 #define CRASH fprintf(stderr, "HID error: pcb called unimplemented STEP function %s.\n", __FUNCTION__); abort()
 
-#define HACK_BOARD_THICKNESS MM_TO_COORD(1.6)
+#define HACK_BOARD_THICKNESS MM_TO_COORD(2.3)
 
 static int step_set_layer (const char *name, int group, int empty);
 static void use_gc (hidGC gc);
